packages/echo-core/tsconfig.json

```diff
@@ -12,16 +12,11 @@
         "moduleResolution": "node",
         "resolveJsonModule": true,
         "isolatedModules": false,
-        "noImplicitAny": false ,
+        "noImplicitAny": false,
         "noEmit": true,
-        "declaration":true,
-        "sourceMap": true,
```

Comment:
After changing to the official rollup typescript plugin (`@rollup/plugin-typescript`) it pointed out some missing settings here, based on the typescript config.

In this particular example: since sourceMap was set to `true` here, we would need to add sourcemap flag to the rollup build as well.

In the end I removed it from here to get around the build error. Do we really want to get source map even for prod build?

Reply:
yes, we really want this in app insight. But it's not working now anyway, so it's ok to remove it here as well for now I think.